@ecomplus/storefront-template/template/scss/components/_header.scss

```diff
@@ -77,6 +103,8 @@
     font-size: var(--font-size-sm);
     line-height: var(--line-height-lg);
     text-transform: uppercase;
+    display: flex;
+    justify-content: space-around;
```

Comment:
acho que você deve adicionar isso só quando for full width mega menu, não?
até então isso vai alterar o menu padrão, e não seria a intenção aqui...